<?xml version="1.0"?>
<launch>
   <arg name="NodeName" default="EvoDCDCController" />
  
    <node name="$(arg NodeName)" pkg="evo_rd_platform_example" type="dcdc_controller_node" output="screen" >
      <param name="loop_rate_hz"       value="10.0"      />
      <param name="can_interface_name" value="USBtin-ID-GOES-HERE" />
      <param name="node_id"            value="10"        />
      <param name="elm_present"        value="1"         />      
    </node>

  </launch>
